### PR TITLE
handle invitation flow for director search admin user#3022

### PR DESCRIPTION
**Pull request by nitheesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/3022

*Description of changes:*
redirection to app after kc signin
redirection after accepting invitation

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/CreateUserProfileLanding.vue [hunk 1]
```diff
@@ -12,7 +12,7 @@
           <v-col lg="8" class="pt-0 pb-0">
             <div class="view-header user-profile-header">
               <h1>Director Search Account Access</h1>
-              <p class="mb-0">You've been invited to administer the '@@replace@@' Director Search Account at the BC Registry. Create your user profile and log in to access this account</p>
+              <p class="mb-0">You've been invited to administer the '----replace----' Director Search Account at the BC Registry. Create your user profile and log in to access this account</p>
```
Comment (sumesh-aot) [inline auth-web/src/components/auth/CreateUserProfileLanding.vue]: The replace text ?

File: auth-web/public/config/configuration.json [hunk 2]
```diff
@@ -4,5 +4,6 @@
   "VUE_APP_AUTH_ROOT_API": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1",
   "VUE_APP_LEGAL_ROOT_API": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1",
   "VUE_APP_STATUS_ROOT_API": "https://status-api-dev.pathfinder.gov.bc.ca/api/v1",
-  "VUE_APP_PATH_NEW_BUSINESS": "https://business-create-dev.pathfinder.gov.bc.ca/businesses/"
+  "VUE_APP_PATH_NEW_BUSINESS": "https://business-create-dev.pathfinder.gov.bc.ca/businesses/",
+  "DIRECTOR_SEARCH_URL": "http://search-web-dev-1rdehl-dev.pathfinder.gov.bc.ca/"
```
Comment (sutherlanda) [inline auth-web/public/config/configuration.json]: Should this be https?

File: auth-web/src/components/auth/CreateUserProfileLanding.vue (hunk 1 above)
Comment (sutherlanda) [inline auth-web/src/components/auth/CreateUserProfileLanding.vue]: Not sure I understand what's happening with the --replace--... is this actually a binding, or a placeholder?

File: auth-web/src/components/auth/CreateUserProfileLanding.vue [hunk 3]
```diff
@@ -12,7 +12,7 @@
...
             </div>
```
Comment (sumesh-aot) [inline auth-web/src/components/auth/CreateUserProfileLanding.vue]: replace text ?

File: auth-web/src/components/auth/mixins/NextPageMixin.vue [hunk 4]
```diff
@@ -50,7 +50,14 @@ export default class NextPageMixin extends Vue {
     switch (this.currentUser?.loginSource) {
       case LoginSource.IDIR:
       case LoginSource.BCROS:
-        return `/${Pages.SEARCH_BUSINESS}`
+        if (this.currentOrganization && this.currentMembership.membershipStatus === MembershipStatus.Active) {
+          if (this.currentMembership.membershipTypeCode === MembershipType.Owner) {
+            return `${Pages.MAIN}/${this.currentOrganization.id}/settings/team-members`
+          } else {
+            return 'director-search-url'
```
Comment (sumesh-aot) [inline auth-web/src/components/auth/mixins/NextPageMixin.vue]: Is it correct, or should we get the director search web url ?

File: auth-web/src/components/auth/mixins/NextPageMixin.vue [hunk 5]
```diff
@@ -50,7 +50,14 @@ export default class NextPageMixin extends Vue {
     switch (this.currentUser?.loginSource) {
       case LoginSource.IDIR:
       case LoginSource.BCROS:
```
Comment (sutherlanda) [inline auth-web/src/components/auth/mixins/NextPageMixin.vue]: You will need to leave the return to SEARCH_BUSINESS for IDIR users.  We don't want to apply the new logic for staff.

File: auth-web/src/components/auth/CreateUserProfileLanding.vue (hunk 1 above)
Comment (nitheesh-aot) [reply auth-web/src/components/auth/CreateUserProfileLanding.vue]: changed that

Comment (nitheesh-aot) [reply auth-web/src/components/auth/CreateUserProfileLanding.vue]: changed that, thats a placeholder for the account name, now its working

File: auth-web/public/config/configuration.json (hunk 2 above)
Comment (nitheesh-aot) [reply auth-web/public/config/configuration.json]: will change that

File: auth-web/src/components/auth/mixins/NextPageMixin.vue (hunk 4 above)
Comment (nitheesh-aot) [reply auth-web/src/components/auth/mixins/NextPageMixin.vue]: we are redirecting to the url from the TermsOfServiceView page.
this text is for checking the condition, whether we redirect to director search url, or contextually push the route for team management page
please let me know if we should need to change this.

File: auth-web/src/components/auth/mixins/NextPageMixin.vue (hunk 5 above)
Comment (nitheesh-aot) [reply auth-web/src/components/auth/mixins/NextPageMixin.vue]: I get that, Thanks, good catch @sutherlanda

File: auth-web/src/components/auth/mixins/NextPageMixin.vue [hunk 6]
```diff
@@ -49,8 +49,17 @@ export default class NextPageMixin extends Vue {
   protected getNextPageUrl (): string {
     switch (this.currentUser?.loginSource) {
       case LoginSource.IDIR:
+        return `${Pages.SEARCH_BUSINESS}`
```
Comment (sutherlanda) [inline auth-web/src/components/auth/mixins/NextPageMixin.vue]: Oops, I think you'll need a slash prepended.  It it works, disregard.

**Pull request merged**